src/api/services/UserAPI.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ */
+
+import * as express from "express";
+import fetch from "node-fetch";
+import querystring = require("querystring");
+import { CONFIG } from "../../Configuration";
+import {
+  ILoginAnonymousResponse,
+  ILoginResponse
+} from "../types/LoginResponse";
+
+// User Service for PagoPA communications
+export class UserAPI {
+  // Login existing user with username and password
+  public static login(
+    res: express.Response,
+    errorCallback: (res: express.Response, errorMsg: string) => void,
+    successCallback: (
+      res: express.Response,
+      loginResponse: ILoginResponse
+    ) => void,
+    username: string,
+    password: string
+  ): void {
+    const url =
+      CONFIG.PAGOPA.HOST +
+      ":" +
+      String(CONFIG.PAGOPA.PORT) +
+      CONFIG.PAGOPA.SERVICES.LOGIN;
+    const queryParams = querystring.stringify({
```

Comment:
remeber that we're not going to have a `username` and `password`

Reply:
Ok, removed into next Push

Reply:
Ok, it's a draft. We'll review it when we'll complete to acquire specs and we'll start to implement related story.